test/format/format_config.c

```diff
@@ -1266,6 +1348,12 @@ config_statistics(void)
 static void
 config_transaction(void)
 {
+    /* Predictable replay requires timestamps. */
+    if (GV(RUNS_PREDICTABLE_REPLAY)) {
+        config_single(NULL, "transaction.timestamps=on", true);
+        config_single(NULL, "transaction.implicit=0", false);
```

Comment:
For predictable replay in test format, specifically, the timestamp drives all decisions about what to do.  Once a thread has a timestamp that it will "work on", it (more or less) seeds its thread-local random number with that timestamp.  That determines exactly what key it will insert/remove/read/update at that timestamp along with any new value.  So we really cannot do predictable replay as designed without having timestamps.